src/utilities/createInitialPermissions.js

```diff
@@ -279,44 +280,64 @@ const permissionsRoles = [
     ],
   },
 ];
+
+
 const createInitialPermissionsFront = async () => {
-  permissionsRoles.forEach(
-    async ({ roleName, permissions, permissionsBackEnd }) => {
-      const allRoles = await Role.find();
+  const userEmail = { email: 'jae@onecommunityglobal.org' };
+  const update = { role: 'Owner' };
+  await User.findOneAndUpdate(userEmail, update);
+
+  const allRoles = await Role.find();
+  const ownerRoleDataBase = allRoles.find(
+    role => role.roleName === 'Owner',
+  );
+
+  let ownerRoleId;
+  let IsAllBackPermissionsOwnerUpdated = true;
+  let IsAllFrontPermissionsOwnerUpdated = true;
+  const updatedOwnerPermissions = {};
+
+  for (let i = 0; i < permissionsRoles.length; i += 1) {
+      const { roleName, permissions, permissionsBackEnd } = permissionsRoles[i];
 
       // Add a new permission if the role Owner has been changed in the  permissionsRoles Array
-      if (roleName === 'Owner') {
-        const ownerRoleDataBase = allRoles.find(
-          role => role.roleName === 'Owner',
-        );
-        const { _id: roleId } = ownerRoleDataBase;
+      if (roleName === 'Owner' && ownerRoleDataBase) {
+        ownerRoleId = ownerRoleDataBase._id;
         const permissionsBackOwnerDataBase = ownerRoleDataBase.permissionsBackEnd;
         const permissionsFrontOwnerDataBase = ownerRoleDataBase.permissions;
-        const IsAllBackPermissionsOwnerUpdated = permissionsBackOwnerDataBase.every(perm => permissions.includes(perm));
-        const IsAllFrontPermissionsOwnerUpdated = permissionsFrontOwnerDataBase.every(perm => permissionsBackEnd.includes(perm));
-
+         IsAllBackPermissionsOwnerUpdated = permissionsBackOwnerDataBase.every(perm => permissions.includes(perm));
+         IsAllFrontPermissionsOwnerUpdated = permissionsFrontOwnerDataBase.every(perm => permissionsBackEnd.includes(perm));
 
         if (!IsAllFrontPermissionsOwnerUpdated) {
-          await Role.findById(roleId, (_, record) => {
-            record.permissions = permissions;
-            record.save();
-          });
+        updatedOwnerPermissions.permissions = permissions;
         }
-        if (!IsAllBackPermissionsOwnerUpdated) {
-          await Role.findById(roleId, (_, record) => {
-            record.permissionsBackEnd = permissionsBackEnd;
-            record.save();
-          });
+        if (!IsAllFrontPermissionsOwnerUpdated) {
+        updatedOwnerPermissions.permissionsBackEnd = permissionsBackEnd;
         }
       }
```

Comment:
These two conditional statements are the same, is the second one supposed to be "!IsAllBackPermissionsOwnerUpdated"?

Reply:
Yess, you are right, already changed to the correct way. Thank you

Reply:
Everything else looks good to me! Going to approve it.